src/libraries/System.Runtime.InteropServices.JavaScript/tests/System.Runtime.InteropServices.JavaScript.UnitTests/System/Runtime/InteropServices/JavaScript/WebWorkerTest.cs

```diff
@@ -439,7 +439,8 @@ public async Task ManagedDelay_ConfigureAwait_True(Executor executor)
         [Theory, MemberData(nameof(GetTargetThreadsAndBlockingCalls))]
         public async Task WaitAssertsOnJSInteropThreads(Executor executor, NamedCall method)
         {
-            using var cts = CreateTestCaseTimeoutSource();
+            CancellationTokenSource? cts = null;
```

Comment:
this is missing `using` now and will trigger in 5 seconds after the test is done

Reply:
cannot use using on nullable variable. I added Dispose at the end instead

Reply:
this will keep hanging when the test fails and add to the confusion.